<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentTab</class>
 <widget class="QWidget" name="ContentTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1487</width>
    <height>1115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QLabel" name="file_name_label">
     <property name="font">
      <font>
       <family>Source Code Pro Black</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(155, 155, 155, 155), stop:1 rgba(225, 225, 225, 155));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QFrame" name="frame">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item alignment="Qt::AlignLeft">
            <widget class="QLabel" name="signals_label">
             <property name="text">
              <string>Signals</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="signals_listView">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item alignment="Qt::AlignRight">
            <widget class="QLabel" name="functions_slots_label">
             <property name="text">
              <string>Functions and Slots</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="functions_slots_listView">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="WidgetListView" name="widget_listView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>5</horstretch>
         <verstretch>5</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="selected_widget_label">
          <property name="maximumSize">
           <size>
            <width>215</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>Selected Widget:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="selected_widget_lineEdit">
          <property name="minimumSize">
           <size>
            <width>215</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>215</width>
            <height>29</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>215</width>
            <height>365</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>215</width>
            <height>365</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="filter_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>215</width>
             <height>311</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Filter</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QComboBox" name="filter_widget_comboBox"/>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Input</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLineEdit" name="filter_input_lineEdit"/>
               </item>
               <item>
                <widget class="QPushButton" name="clear_filter_input_pushButton">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>clear</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="filter_by_name_radioButton">
                 <property name="text">
                  <string>by Name</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="filter_by_widget_radioButton">
                 <property name="text">
                  <string>by Widget</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="sort_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>93</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Sort</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="sort_by_comboBox">
              <property name="toolTip">
               <string>sort by</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sort_direction_comboBox">
              <property name="toolTip">
               <string>direction</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sort_widgets_pushButton">
              <property name="text">
               <string>Sort</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetListView</class>
   <extends>QListView</extends>
   <header>pyqtsocius.ui_elements.views.widget_listview</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pyqtsocius/ui_elements/pyqt_sorter_ressources.qrc"/>
 </resources>
 <connections/>
</ui>
